### ISSUE TITLE
Could you please specify the usage of your code? THX

### ISSUE THREAD

**Issue by JustinAsdz** [Elin24/PSPL#3]
Hi  Elin

Thanks for your work, recently I want to reproduce the result of your method. Can you specify the work-flow of your paper, for convenient usage for others to use.

Thanks !

**Comment by Elin24** [comment]
There are 6 steps for an input **LR** image and its ground truth **HR**:
1. use a super-resolution network to generate **SR** image;
2. figure out the SSIM map between HR and SR, corresponding to the following code:
```python
ssim = pytorch_ssim.ssim(hr, sr, reduction='none').detach()
```
3. use the gaussian function to generate attention map:
```python
# the meaning of sigma and maxVal can be found in my paper
gauss = lambda x: torch.exp(-((x+1) / sigma) ** 2) * self.maxVal
weight = gauss(ssim).detach()
```
4. generate new HR and new SR image:
```python
nsr = sr * weight
nhr = hr * weight
```
5. adopt loss function like mse or mae to calculate final loss value:
```python
loss = mae(nsr, nhr) # use mae as example
```
6. back propagation, which can be done by Pytorch use `loss.backward()`

---

PSPL is a very sample method to  accelerate convergence, all its function can be represented by the **code show in [Core Part](https://github.com/Elin24/PSPL#core-parts)** in README file.

**Comment by JustinAsdz** [comment]
Thanks for your quick reply,

Your Explanation is clear and percise, but can you show the steps to run the code
for example : about dataset preparation or running file order.
Thanks

**Comment by Elin24** [comment]
I have explained it in **[issue 1](https://github.com/Elin24/PSPL/issues/1)**, including preparing data and running the code in *src*. 
You can refer to it.

**Comment by JustinAsdz** [comment]
Thanks ,go to issue 1
